frontend/packages/pipelines-plugin/src/components/pipelines/modals/triggers/AddTriggerModal.tsx

```diff
@@ -48,6 +51,7 @@ const AddTriggerModal: React.FC<AddTriggerModalProps> = ({ pipeline, close }) =>
       initialValues={initialValues}
       onSubmit={handleSubmit}
       validationSchema={addTriggerSchema(t)}
+      enableReinitialize
```

Comment:
Hmmm, not sure about this... kinda would want to wait for the data before rendering Formik. Reinitializing data could have side-effects mid-usage.

Reply:
I have removed the `enableReInitiaze` prop and waiting for the data to render Formik.